### PR TITLE
Migrate iOS application to Compose Multiplatform

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

File: shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/SessionDetailed.kt [hunk 1]
```diff
@@ -33,9 +32,9 @@ fun SessionDetailed(
     isAWS: Boolean,
     controller: AppController
 ) {
-    val scaffoldState = rememberBottomSheetScaffoldState(
-        bottomSheetState = BottomSheetState(BottomSheetValue.Expanded)
-    )
+//    val scaffoldState = rememberBottomSheetScaffoldState(
```
Comment (rsinukov) [inline shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/SessionDetailed.kt:35]: same here

File: shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/components/Feedback.kt [hunk 2]
```diff
@@ -87,7 +86,8 @@ fun FeedbackForm(onSend: (String) -> Unit, onClose: () -> Unit) {
                     .fillMaxWidth()
                     .height(200.dp)
                     .onKeyEvent {
-                        if (it.nativeKeyEvent.keyCode != android.view.KeyEvent.KEYCODE_ENTER) return@onKeyEvent false
+                        // TODO
```
Comment (rsinukov) [inline shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/components/Feedback.kt:89]: todo

**Review by rsinukov — APPROVED** [review]
there are some comments and todos left, please check

File: androidApp/src/main/kotlin/org/jetbrains/kotlinconf/android/MainActivity.kt [hunk 3]
```diff
@@ -1,43 +1,26 @@
...
-            "https://kotlin-conf-staging.labs.jb.gg/"
+        val context = ApplicationContext(
+            application,
+            R.mipmap.ic_launcher,
         )
-
-        fun addOnBackCallback(block: OnBackPressedCallback) {
-            onBackPressedDispatcher.addCallback(this, block)
-        }
+//        fun addOnBackCallback(block: OnBackPressedCallback) {
```
[the remark this answers is not in the document]
Comment (e5l) [reply androidApp/src/main/kotlin/org/jetbrains/kotlinconf/android/MainActivity.kt:18]: thanks, will drop

**Pull request merged**